build.gradle

```diff
@@ -0,0 +1,70 @@
+group = 'org.sagebionetworks'
+ext.name = 'bridge-base'
+version = '0.1.0+' + (new Date()).format(
+        'yyyyMMddHHmm', TimeZone.getTimeZone("UTC"))
+
+/*** Java ***/
+
+apply plugin: 'java'
+
+sourceCompatibility = '1.7'
+targetCompatibility = '1.7'
```

Comment:
Does Play support Java 8 yet? Because Java 8 has some nice syntactical sugar (lambdas) that I'd like to use.